safe/definitions/fields.py

```diff
@@ -2249,3 +2227,147 @@
     over_60_displaced_count_field,
     disabled_displaced_count_field
 ] + minimum_needs_fields
+
+
+# Productivity field
+productivity_rate_field = {
+    'key': 'productivity_rate_field',
+    'name': tr('Productivity Rate'),
+    'field_name': 'productivity_rate',
+    'type': qvariant_numbers,
+    'length': default_field_length,
+    'precision': 0,
+    'absolute': True,
+    'help_text': tr('The rate of productivity of a crop in Kwt/Ha unit.'),
```

Comment:
Hi @Gustry 
I have added the explanation in the last commit. I am not sure it's the best. May be we should ask @fredychandra or @samnawi to get a better explanation.